protocol/v2/qbft/instance/instance.go

```diff
@@ -162,7 +161,14 @@ func (i *Instance) BaseMsgValidation(msg *specqbft.SignedMessage) error {
 		return errors.Wrap(err, "invalid signed message")
 	}
 
-	if msg.Message.Round < i.State.Round {
+	// If a node gets a commit quorum before round change and other nodes get it after,
+	// then the node with quorum before wouldn't be able to receive next round messages from others
+	// due to slashing protection, unless we allow decided messages from previous round.
+	if decided := msg.Message.MsgType == specqbft.CommitMsgType && len(msg.Signers) > 1; decided {
```

Comment:
I think we should use `len(msg.Signers) >= quorumSize`

Reply:
I agree, done